core/src/main/java/org/infinispan/configuration/cache/TakeOfflineConfigurationBuilder.java

```diff
@@ -81,9 +72,4 @@ public TakeOfflineConfigurationBuilder read(TakeOfflineConfiguration template) {
       this.attributes.read(template.attributes());
       return this;
    }
-
-   @Override
-   public String toString() {
-      return this.getClass().getSimpleName() + attributes;
```

Comment:
is this intentional?

Reply:
As I was going through removing all of the empty `validate()` methods, I also got carried away removing useless method overrides on the builder objects (`hashCode`/`equals`/`toString`) unless strictly necessary